backend/flindt/feedback/models.py

```diff
@@ -164,8 +164,33 @@ def send_feedback_received_message():
             messenger.send_message(message)
 
         def send_rating_received_message():
-            message = _('{} {} just rated the feedback that you gave.'.format(self.recipient.first_name,
-                                                                              self.recipient.last_name))
+            """
+            When the feedback that a user (sender) has given is rated by
+            the recipient, a message is sent with the content of the rating
+            and a link to Flindt.
+            """
+            def type_of_feedback():
+                # Determines whether the rated feedback is on a role or individual.
+                if self.role:
+                    return 'role'
+                if self.individual:
+                    return 'personal'
+
+            message = _('{first_name} {last_name} just rated the feedback that you gave. ' \
```

Comment:
The backslashes are not necessary between brackets.
Also extra space after the .